video-img.py

```diff
@@ -0,0 +1,65 @@
+import cv2
+import time
+import numpy as np
+import os
+
+
+
+def save_img(video_path, result_path, interval_num, resize_wh, rot90=False, show=False):
+    '''
+    将视频每帧保存到文件夹，命名格式：视频名_时间戳_num
+    Save each frame of the video to a folder, naming format: video name_timestamp_num
+    :param video_path: 视频路径
+    :param result_path: 结果保存路径
+    :param interval_num: 每间隔几帧保存一帧 (Save a frame every few frames)
+    :param resize_wh: 指定保存图片的长宽 (Specifies the length and width of the saved image)
```

Comment:
We will work only with horizontal videos. Video resolution is usually caracterized only by its height (e.g. [1080p)](https://en.wikipedia.org/wiki/1080p). Because of that, I think it would be better to ask for a **target height**.
For instance, if we feed a 368p video to this function, we could specify `target_height=1080` to resize the extracted frames to 1080 pixels in height, and the width calculated so that we have the same aspect ratio in the end.